File: Website/src/app/domain/models/saque.ts
export class Saque{
    id : number = 0;
    dataHora: string = "";
    valor: number = 0;
    descricao: string = "";
    carteiraId: number = 0;

    constructor(id:number, dataHora: string, valor: number, descricao: string, carteiraId: number) {
        //specify your own constructor logic
        this.id = id;
        this.dataHora = dataHora;
        this.valor = valor;
        this.descricao = descricao;
        this.carteiraId = carteiraId;
    }
}

File: API/ContaCentral/ContaCentral/Domain/Models/Saque.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ContaCentral.Domain.Models
{
    public class Saque
    {
        public int Id { get; set; }
        public decimal Valor { get; set; }
        public DateTime DataHora { get; set; }
        public string Descricao { get; set; }

        public int CarteiraId { get; set; }
        [ForeignKey("CarteiraId")]
        public Carteira Carteira { get; set; }
    }
}
